### PR TITLE
feat: add crvBUSD and crvCOMP v2 migration

### PR DESCRIPTION
**Pull request by patcito**

### CODE REVIEW

File: app/containers/Vaults/migrationWhitelist.json [hunk 1]
```diff
@@ -94,5 +94,21 @@
     "vaultTo": "0xa9fE4601811213c340e850ea305481afF02f5b28",
     "apyTooltip": "Please migrate funds to our new vault to continue earning yield.",
     "migrationMessage": "This vault is no longer active. Withdraw funds or migrate your balance to the new v2 WBTC yVault to continue earning yield."
+  },
+  {
+    "symbol": "crvBUSD",
+    "token": "0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B",
+    "vaultFrom": "0x2994529C0652D127b7842094103715ec5299bBed",
```
Comment (x48-crypto) [inline app/containers/Vaults/migrationWhitelist.json:101]: @dudesahn can you double check these are the correct addresses?

Comment (patcito) [reply app/containers/Vaults/migrationWhitelist.json:101]: @x48-crypto vault from was confirmed by @fameal. Maybe you can confirm tokens too @fameal

**Comment by patcito** [comment]
- Found address [0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B](https://etherscan.io/address/0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B) in files . This address is from token `yDAI+yUSDC+yUSDT+yBUSD`  
- Found address [0x2994529C0652D127b7842094103715ec5299bBed](https://etherscan.io/address/0x2994529C0652D127b7842094103715ec5299bBed) in files . This address is from vault `yyDAI+yUSDC+yUSDT+yBUSD`  
- Found address [0x8ee57c05741aA9DB947A744E713C15d4d19D8822](https://etherscan.io/address/0x8ee57c05741aA9DB947A744E713C15d4d19D8822) in files . This address is from vault `yvCurve-BUSD`  
- Found address [0x845838DF265Dcd2c412A1Dc9e959c7d08537f8a2](https://etherscan.io/address/0x845838DF265Dcd2c412A1Dc9e959c7d08537f8a2) in files . This address is from token `cDAI+cUSDC`  
- Found address [0x629c759D1E83eFbF63d84eb3868B564d9521C129](https://etherscan.io/address/0x629c759D1E83eFbF63d84eb3868B564d9521C129) in files . This address is from vault `yvcDAI+cUSDC`  
- Found address [0xD6Ea40597Be05c201845c0bFd2e96A60bACde267](https://etherscan.io/address/0xD6Ea40597Be05c201845c0bFd2e96A60bACde267) in files . This address is from vault `yvCurve-Compound`

**Comment by dudesahn** [comment]
submitted a commit to update naming, other than that lgtm

**Review by dudesahn — APPROVED** [review]

File: app/containers/Vaults/migrationWhitelist.json (hunk 1 above)
Comment (dudesahn) [reply app/containers/Vaults/migrationWhitelist.json:101]: Confirmed, addresses are good

**Review by x48-crypto — APPROVED** [review]

**Pull request merged**